modules/extension/brewer/src/main/java/org/geotools/styling/builder/ContrastEnhancementBuilder.java

```diff
@@ -49,18 +53,38 @@ public ContrastEnhancementBuilder gamma(double gamma) {
         return gamma(literal(gamma));
     }
 
-    public ContrastEnhancementBuilder normalize() {
-        this.method = ContrastMethod.NORMALIZE;
-        this.unset = false;
-        return this;
+    public ContrastEnhancementBuilder normalize(Map<String, Expression> constrastParameters) {
+        return contrastMethod("normalize", constrastParameters);
```

Comment:
Not sure using a Map as a parameter makes this easier ... would do better to have several methods with explicit documented parameters so developers do not have to remember keys.

```
normalize( new KVP("Algorithm",ff,literal("StretchToMinimumMaximum"),"minValue",ff,literal(0,),"maxValue",ff.literal(200)))
```

Does not seem much fun.